guide/principles/principles.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: guide
+title: Principles
+description: The key principles to follow when designing Bitcoin products.
+nav_order: 9
+parent: Foundations
+permalink: /guide/foundations/principles/
+main_classes: -no-top-padding
+image: /assets/images/guide/foundations/principles/page-principles.jpg
+---
+
+<!--
+
+Editor's notes
+
+The key principles of designing for Bitcoin
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/foundations/principles/principles.jpg"
+   retina = "/assets/images/guide/foundations/principles/principles@2x.jpg"
+   mobile = "/assets/images/guide/foundations/principles/principles-mobile.jpg"
+   mobileRetina = "/assets/images/foundations/principles/principles-mobile@2x.jpg"
+   alt-text = "Principles header illustration"
```

Comment:
Alt text should describe the image content (which is a bit hard in this case due to the minimalism).

Reply:
Improved